corehq/apps/users/views/__init__.py

```diff
@@ -761,6 +761,10 @@ class InviteWebUserView(BaseManageWebUserView):
     @memoized
     def invite_web_user_form(self):
         role_choices = UserRole.role_choices(self.domain)
+        loc = None
+        if 'location_id' in self.request.GET:
+            from corehq.apps.locations.models import SQLLocation
+            loc = SQLLocation.objects.get(location_id=self.request.GET.get('location_id'))
```

Comment:
might want to use `get_object_or_404` in case it gets deleted or something odd like that